vmm/service/kuasar-vmm

```diff
@@ -0,0 +1,5 @@
+# configure the vmm-sandboxer command options
+OPTIONS='--listen /run/vmm-sandboxer.sock --dir /run/kuasar-vmm'
+
+# set vmm-sandboxer log level
+RUST_LOG=info
```

Comment:
I want to provide two ways to configure the vmm-sandboxer process's log level. In this place, keep the original way to set the log level by `RUST_LOG` environment which is set by default.
Another way in the #36 provides to configure log level in the `config_<hypervisor>.toml` file which is more flexible for user to change the log level.

#36 supports log_level set in the `config_<hypervisor>.toml` file overrides the default log  level set in the `/etc/sysconfig/kuasar-vmm` environment file.

Reply:
it  seems the env of `RUST_LOG` is enough to set the log level and it is simple. keeping these to ways to set the log level makes things complicated, I am wondering if we have to revert the #36 ?

Reply:
It does seem a bit repetitive, and I think it's better to keep the support for configuring it in the config file rather than in an systemd service env file.
So, we don't need to revert the #36, just remove the `RUST_LOG` env option in the `/etc/sysconfig/kuasar-vmm` env file.